doc/source/cluster/cluster_under_construction/ray-clusters-on-vms/user-guides/configuring-autoscaling.rst

```diff
@@ -9,11 +9,8 @@ This page provides an overview of the parameters for the Ray autoscaler. A Ray c
 the number of nodes based on the resources requested by the task, actor, or placement group. If there is insufficient resource to serve
 the request they will be put into pending and will not execute. The autoscaler runs a bin-packing algorithm to determine the list of nodes
 the cluster should use to satisfy the resource requests, and adds / removes nodes from the cluster to maximize the cluster utilization
-while providing sufficient resource to execute the requests.
-
-.. note::
-  The autoscaler currently only looks at the amount of cpu and gpu requested and adds / removes nodes accordingly. It does not take into
-  the account the memory request nor the disk usage.
+while providing sufficient resource to execute the requests. The autoscaler currently only looks at the amount of cpu and gpu requested
```

Comment:
It's possible to use memory as a logical resource for autoscaling.
One can also annotate nodes with "custom resource" capacities -- arbitrary pairs like
`{"thingy": 10}` and then schedule tasks with `@ray.remote(resources={"thingy": 1})`.

Reply:
What we're trying to express here is that the autoscaler looks at logical resources request and not physical resource usage.

We can remove "currently" -- I don't think we plan to change this scheme.

Reply:
I think some phrasing like "The autoscaler only considers logical resources (e.g. those specified in `ray.remote` and displayed in `ray status`), with the exception of `object_store_memory` which it does not consider.

Reply:
sg